lib/fathead/gradle_docs/parse_util.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
can you proper format this file? That way will be more easy to read and review :)